### PR TITLE
added  the user name

### LINKED ISSUE #1: Add the user name in the index file
username: **CrowTi**

### PR DESCRIPTION
**Pull request by CrowTi**
Check Please =)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,6 +12,7 @@
   </head>
   <body>
     <h1>Hello, world!</h1>
+    <h2>CrowTi</h2>
```
Comment (DocenciaJP) [inline index.html:15]: That it OK....!!!

Comment (sem-hm) [inline index.html:15]: the username is ok

File: index.html [hunk 2]
```diff
@@ -12,6 +12,7 @@
   </head>
```
Comment (sem-hm) [inline index.html:12]: ok

Comment (DocenciaJP) [inline index.html:12]: it is complete..!!!

**Review by sem-hm — APPROVED** [review]
LGTM

**Review by DocenciaJP — APPROVED** [review]
LGTM

**Review by 1382308 — APPROVED** [review]
LGTM

**Pull request merged**